src/lambdas/decorate-pools.ts

```diff
@@ -13,10 +13,10 @@ export const handler = async (): Promise<any> => {
     log("Loading Tokens")
     const tokens = await getTokens();
     log(`Loaded ${tokens.length} tokens. Loading Pools`) 
-    const pools = await getPools();
+    const pools = await getPools(1);
     log(`Decoracting ${pools.length} Pools`)
     const decorateStartTime = Date.now();
-    const poolDecorator = new PoolDecorator(pools);
+    const poolDecorator = new PoolDecorator(pools, 1);
```

Comment:
Is that the network you're passing in? Will you change that to be a variable?

Reply:
yea, that was just temporary while testing. There is one lambda per network now.